rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketController.java

```diff
@@ -377,4 +386,19 @@ private void addAttributeFilter(WorkbasketQuery query,
             }
         }
     }
+
+    private PageMetadata initPageMetadata(String pagesizeParam, String pageParam, long totalElements)
+        throws InvalidArgumentException {
+        long pagesize;
```

Comment:
TYPO: use camecase notation pageSize

Reply:
I took this from an example. I believe you're right.